index.html

```diff
@@ -326,29 +326,29 @@ <h3 class="skills-heading">
         </section>
 
         <!-- ======================== Contact Page ======================== -->
-        <a name="contact-page"></a>
         <section class="contact-form-page">
+            <a name="contact-page"></a>
             <div class="contact-form-background"></div>
             <img src="./Images/1ellipse.png" alt="" class="ellipse-image">
             <img src="./Images/2upRectangle.png" alt="" class="up-rectangle">
             <img src="./Images/3bluecircle.png" alt="" class="blue-circle">
             <img src="./Images/4downRectangle.png" alt="" class="down-rectangle">
             <img src="./Images/5whiteEllipse.png" alt="" class="white-circle">
-            <h2 class="contact-form-heading">Contact me</h2>
+            <h1 class="contact-form-heading">Contact me</h1>
```

Comment:
- [ ] Great job applying those accessibility best practices to your project! 💯 However, I suggest you revert this change. Ideally, you would want to have only one `<h1>` element for each page, this tag is reserved for the title that describes better the content of the page (The heading one which you already changed). It serves as an index for search engines to find your page, so please consider reverting this change! 🤓 🔧